cdap-ui/app/directives/widget-container/widget-container.less

```diff
@@ -1,6 +1,6 @@
 .my-widget-container {
   .form-control {
-    width: 90%;
+    // width: 90%;
```

Comment:
Can we just remove this? With the full width, we don't need to reduce the size of the input elements right?